From f3ec52ad9c039de6bf0c1904f17171c130d0f713 Mon Sep 17 00:00:00 2001
From: Shawn Guo <shawn.guo@freescale.com>
Date: Fri, 13 Jun 2014 10:41:19 +0800
Subject: [PATCH] ENGR00317981: drm: vivante: drm_fasync is gone
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The drm_fasync() is gone since commit b0e898ac555e (drm: remove FASYNC
support).  Remove it from vivante driver to fix the build error below.

  CC      drivers/gpu/drm/vivante/vivante_drv.o
drivers/gpu/drm/vivante/vivante_drv.c:68:12: error: ‘drm_fasync’ undeclared here (not in a function)

Signed-off-by: Shawn Guo <shawn.guo@freescale.com>
---
 drivers/gpu/drm/vivante/vivante_drv.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/vivante/vivante_drv.c b/drivers/gpu/drm/vivante/vivante_drv.c
index 4caf46f..d64dfa8 100644
--- a/drivers/gpu/drm/vivante/vivante_drv.c
+++ b/drivers/gpu/drm/vivante/vivante_drv.c
@@ -65,7 +65,6 @@ static const struct file_operations viv_driver_fops = {
 	.unlocked_ioctl = drm_ioctl,
 	.mmap = drm_mmap,
 	.poll = drm_poll,
-	.fasync = drm_fasync,
 	.llseek = noop_llseek,
 };
 
-- 
1.8.0

